Source/Assets/PlayFabEditorExtensions/Editor/Scripts/Panels/PlayFabEditorPackages.cs

```diff
@@ -0,0 +1,89 @@
+using UnityEditor;
+using UnityEngine;
+using System;
+using System.Reflection;
+
+namespace PlayFab.PfEditor
+{
+    public class PlayFabEditorPackages : Editor
+    {
+        private const int buttonWidth = 150;
+
+        public static bool IsPubSubInstalled { get { return GetIsPubSubInstalled(); } }
+
+        public static void DrawPackagesMenu()
+        {
+            var labelStyle = new GUIStyle(PlayFabEditorHelper.uiStyle.GetStyle("label"));
+            if(!IsPubSubInstalled)
+            {
+                GUILayout.BeginHorizontal();
+                GUILayout.Label("PersistentSockets: ");
+                if (GUILayout.Button("Install", PlayFabEditorHelper.uiStyle.GetStyle("Button"), GUILayout.MaxWidth(buttonWidth), GUILayout.MinHeight(32)))
+                {
+                    string possibleNewtonsoftPath = "";
+                    if(GetIsNewtonsoftInstalled(out possibleNewtonsoftPath))
+                    {
+                        EditorUtility.DisplayDialog("Newtonsoft is already installed.",
+                            "Please delete your version of Netwonsoft.json.dll in \n\n" + possibleNewtonsoftPath + " \n and retry the install.\n\n compiler conflicts will happen if this stays around", "Continue", "Cancel");
+                    }
+                    else
+                    {
+                        ImportPubSubSDK();
+                    }
+                }
+
+                GUILayout.EndHorizontal();
+            }
+            else
+            {
+                EditorGUILayout.LabelField("PersistentSockets is already Installed", labelStyle, GUILayout.MinWidth(EditorGUIUtility.currentViewWidth));
+            }
+        }
+
+        public static void ImportPubSubSDK()
+        {
+            // TODO: when the PR is checked in, we'll need to use the below url on api.playfab.com 
+            // "https://github.com/PlayFab/UnitySDK/tree/master/Packages/PubSub.unitypackage";
```

Comment:
Remote these comments, as they're already handled by the playfab redirect link below.